README.md

```diff
@@ -556,14 +556,18 @@ python manage.py load_advisory_opinions [-f FROM_AO_NO]
 python manage.py load_current_murs [-f FROM_MUR_NO]
 ```
 
-#### Loading all legal documents for the 1st time
+#### Loading all current legal documents for the 1st time (excludes archived MURs)
 ```
-python manage.py reinitialize_all_legal_docs
+python manage.py initialize_current_legal_docs
+```
+#### Loading all archived MURs for the 1st time
+```
+python manage.py create_archived_murs_index
```

Comment:
also add docs for `load_archived_murs`

Reply:
The headings say "Loading..." while the commands are actually for _initializing_ indexes.

Reply:
We should also add docs for loading _archived_murs_.